assets/src/hooks/useDetour.ts

```diff
@@ -28,8 +28,12 @@ export const useDetour = (input: UseDetourInput) => {
   // Record snapshots when changed
   useEffect(() => {
     const snapshotSubscription = actorRef.subscribe(() => {
-      const serializedSnapshot = JSON.stringify(actorRef.getPersistedSnapshot())
+      const persistedSnapshot = actorRef.getPersistedSnapshot()
+      const serializedSnapshot = JSON.stringify(persistedSnapshot)
       localStorage.setItem("snapshot", serializedSnapshot)
+      if (persistedSnapshot.context.uuid) {
+        putDetourUpdate(persistedSnapshot.context)
+      }
```

Comment:
I might as well do it here 👍

Reply:
Actually! I'd rather not do that transition, because I'd also need to make sure that we read from the database instead of localStorage, and that just feels like more than this PR should do.